pkg/ccl/multitenantccl/tenantcostclient/tenant_side.go

```diff
@@ -299,6 +299,8 @@ func (c *tenantSideCostController) updateRunState(ctx context.Context) {
 		ru += float64(deltaPGWireEgressBytes) * float64(c.costCfg.PGWireEgressByte)
 	}
 
+	// KVRUs are not explicitly included here as they are alreay accounted for
```

Comment:
Typo: alreay. How about the following instead?

```
// KV RUs are not included here, these metrics correspond only to the SQL pod.
```

Reply:
Done.